README.md

```diff
@@ -27,16 +26,17 @@ For applications integrating the nRF70 Series BM driver it is recommended to use
     cd nrf70-bm
     west update
     ```
+3. Prepare the development environment based on https://docs.zephyrproject.org/latest/develop/getting_started/index.html
```

Comment:
It would be nice if we point to the "command-line" version of preparing the toolchain and the environment, since your instructions below are related to that.